controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -91,6 +96,10 @@ spec:
             initialDelaySeconds: 15
             periodSeconds: 10
             timeoutSeconds: 15
+          ports:
+          - containerPort: {{ include "cloud-controller-manager.port" . }}
```

Comment:
Can you add/define this? It's not yet part of this chart

Reply:
https://github.com/gardener/gardener-extensions/blob/6c6f41e20857c06ffb07488ea952485e7870f015/controllers/provider-openstack/charts/internal/cloud-controller-manager/templates/_helpers.tpl#L7-L13